tfhe/src/high_level_api/integers/oprf.rs

```diff
@@ -0,0 +1,101 @@
+use super::{FheIntId, FheUintId};
+use crate::high_level_api::global_state;
+use crate::high_level_api::keys::InternalServerKey;
+use crate::integer::oprf::SignedRandomizationSpec;
+use crate::{FheInt, FheUint};
+use concrete_csprng::seeders::Seed;
```

Comment:
use crate::core_crypto::commons::math::random::Seed

it's a re-export of the cpsrng type, if you have an idea to better re-export types, let me know